docs/website/docs/development/architecture/podman-limitations.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Podman limitations
+sidebar_position: 40
+---
+
+The `odo dev` command is able to work either on Podman or on a Kubernetes cluster. 
+
+The motivation behind the support for the Podman platform is to lower the learning curve
+for developers working on containerized applications. As a matter of fact, Podman is simpler
+to apprehend, install and maintain than a Kubernetes cluster.
+
+Thanks to the support for the **Kubernetes Pod** abstraction by Podman, `odo`, and 
+the user, can work on both Podman and Kubernetes on top of this abstraction.
+
+Here are a list of limitations when `odo` is working on Podman:
+
+## Apply command are not supported
```

Comment:
`apply` commands referencing an `Image` component are supported (since https://github.com/redhat-developer/odo/pull/6577). So I'd suggest being more specific here, maybe with something like this?
```suggestion
## Apply commands referencing Kubernetes or OpenShift Components are not supported
```